include/deal.II/grid/grid_out.h

```diff
@@ -891,6 +893,12 @@ namespace GridOutFlags
      * Output co-faces/edges.
      */
     bool output_co_faces;
+
+    /**
+     * Output only faces/co-faces, which are relevant (and differ
```

Comment:
The English language is less regular about the comma -- it's really more of a tool to break a train of thought (except in enumerations) and not necessary here.
```suggestion
     * Output only faces/edges which are relevant (and differ
```